cclib/parser/turbomoleparser.py

```diff
@@ -208,6 +227,51 @@ def extract(self, inputfile, line):
             self.set_attribute('vibirs', vibirs)
             self.set_attribute('vibdisps', vibdisps)
 
+        # In this section we are parsing mocoeffs and moenergies from
+        # the files like: mos, alpha and beta.
+        # $scfmo    scfconv=6   format(4d20.14)
+        # # SCF total energy is     -382.3457535740 a.u.
+        # #
+        #      1  a      eigenvalue=-.97461484059799D+01   nsaos=60
+        # 0.69876828353937D+000.32405121159405D-010.87670894913921D-03-.85232349313288D-07
+        # 0.19361534257922D-04-.23841194890166D-01-.81711001390807D-020.13626356942047D-02
+        # ...
+        # ...
+        # $end
+        if (line.startswith('$scfmo') or line.startswith('$uhfmo')) and line.find('scfconv') > 0:
+            if line.strip().startswith('$uhfmo_alpha'):
+                self.unrestricted = True
+
+            while not (line.strip().startswith('1') and line.find('eigenvalue') > 0):

```

Comment:
`startswith('1')` should not be there; that's the 1-based index of the MO.

Reply:
Yeah. I understood that after seeing the `water` mos file. Will make the changes.